np-gravity.py

```diff
@@ -87,7 +88,7 @@ def draw(screen, game):
 
 # create game instance of the correct type
 puzzle = Puzzle(raw_input)
-game = Megalit(puzzle) if args.megalit else Popils(puzzle)
+game = Megalit(puzzle) if not args.megalit else Popils(puzzle)
```

Comment:
```suggestion
game = Megalit(puzzle) if args.megalit else Popils(puzzle)
```
I think this was correct the old way. If I run np-gravity w/ no parameters right now, it builds a megalit level (but the window title says "Popils Reduction"

Reply:
good catch, i just changed it for testing purposes